### PR TITLE
Implementing Hierarchy Section for New Home Page

### PR DESCRIPTION
**Pull request by jouypo23**
Completes [Story 12323](https://app.shortcut.com/clarkcan/story/12323/implement-learning-outcomes-and-hierarchies-sections).

Note: 
- This PR still needs the official verbiage for the two sections, those sections will be recommitted once verbiage is available.
- The learning object hierarchy used is different from the Figma mock to ensure a more relevant example.

### Screenshots
Desktop
Screen Shot 2022-08-11 at 3 24 17 PM

Mobile
Screen Shot 2022-08-11 at 1 18 34 PM
Screen Shot 2022-08-11 at 1 19 04 PM

### CODE REVIEW

**Review by JosueMolinaMorales — APPROVED** [review]

File: src/app/cube/new-home/learning-object-info/hierarchies/hierarchies.component.html [hunk 1]
```diff
@@ -1 +1,27 @@
...
+    <h1>Leverages the Bloom's taxonomy for measureable results</h1>
+    <!-- Dummy text until the full text is provided for this section -->
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    <br>
+    <h1>Related curricula grouped hierarchically</h1>
+    <!-- Dummy text until the full text is provided for this section -->
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```
Comment (EVogel1999) [inline src/app/cube/new-home/learning-object-info/hierarchies/hierarchies.component.html:9]: We need to replace the lorem ipsums with real descriptions.  I'd ask Paige or Mike about this.

**Review by EVogel1999 — COMMENTED** [review]
Just one comment on lorem ipsum.  Looks good!

**Review by pmanal1 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]
This looks good, we can swap in the lorem ipsum later on after we get everything merged together.

**Review by EVogel1999 — APPROVED** [review]
After talking with Paige, the Lorem Ipsums will be replaced later.  Looks good!

File: src/app/cube/new-home/learning-object-info/hierarchies/hierarchies.component.ts [hunk 2]
```diff
@@ -1,15 +1,26 @@
...
+import { Component, OnInit, Input} from '@angular/core';
+import { ClientRequest } from 'http';
 
 @Component({
   selector: 'clark-hierarchies',
   templateUrl: './hierarchies.component.html',
   styleUrls: ['./hierarchies.component.scss']
 })
 export class HierarchiesComponent implements OnInit {
+  @Input() length: 'nanomodule' | 'micromodule' | 'module' | 'unit' | 'course';
```
Comment (mfranz1) [inline src/app/cube/new-home/learning-object-info/hierarchies/hierarchies.component.ts:10]:
```suggestion
  length: 'nanomodule' | 'micromodule' | 'module' | 'unit' | 'course';
```
This variable isn't being dynamically set so we don't have to use the @Input decorator

**Review by mfranz1 — CHANGES_REQUESTED** [review]

**Review by mfranz1 — APPROVED** [review]
Nice work!

**Pull request merged**